### DISCUSSION TITLE
V0.16.4 shows Gbps instead of Mbps

### DISCUSSION THREAD

**Discussion by seh2000** [alexjustesen/speedtest-tracker#1192] [Q&A] [answered]
Just - for me maybe - a small issue.
V0.16.4 now shows the speed in Gbps instead of in Mbps.
Is it possible to change somewhere to Mbps?

**Comment by alexjustesen** [comment, marked as answer]
https://github.com/alexjustesen/speedtest-tracker/issues/1186#issuecomment-1954562242
